.github/workflows/_tests.yml

```
@@ -0,0 +1,36 @@
+name: Run tests
```

```suggestion
name: Run notebook tests
```

I think this file is just notebook tests, correct?